content/api/entity-reference/gift/index.html

```diff
@@ -1,62 +1,30 @@
 ---
 layout: layout-container
-name: Gift Entity Reference
-title: Gift Entity Reference
+name: Gift Entity Reference (Beta)
+title: Gift Entity Reference (Beta)
 order: 50
-published: false
-showInNav: false
+published: true
+showInNav: true
 back_to_top: true
 showBreadcrumbs: true
 ---
 
-<h1>{{ name}}</h1>
+<h1>{{ name }}</h1>
 
-<p>Updated: January 26, 2017</p>
+<p>Updated: March 8, 2017</p>
 
 
     <p>This topic describes the entity and type representations for common items used that the Gift API uses. To learn about how to set up your developer account and work with the SKY API, see the <a href="{{ stache.config.guide_getting_started }}">Getting Started</a> guide.</p>
```

Comment:
Looks like we're using the stache config here for the getting started page... We should be consistent